lib/entities/npcs/animals/piggy.dart

```diff
@@ -124,6 +124,8 @@ class Piggy extends NPC {
 		StatManager.add(email, Stat.piggies_nibbled);
 		SkillManager.learn(SKILL, email);
 		nibbleCounts[email] = (nibbleCounts[email] ?? 0) + 1;
+		messageBus.publish(new RequirementProgress('piggyNibble', email));
+		QuestEndpoint.questLogCache[email].offerQuest('Q11, Piggy Nibbler Quest');
```

Comment:
I had just meant a comment in the code about which quest this was so that people who are reading it know. This call is going to search for a quest with an id of "Q11, Piggy Nibbler Quest" but you're defining it as "Q11" so this is going to fail.